common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
number of characters shorter :trollface:

`import scalaz.syntax.std.option._ xx.some xx.some`
`Some(xx) Some(xx)`
`Option(xx) Option(xx)`